packages/docs-site/src/components/presenters/data-table/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import sortBy from 'lodash/sortBy'
+
+import './data-table.scss'
+
+import TableHead from './table-head'
+import TableBody from './table-body'
+
+function extractHeadings(data) {
+  return data.length > 1 ? Object.keys(data[0]) : []
+}
+
+const DataTable = ({ data, caption }) => {
+  const [tableData, setTableData] = useState(data)
+
+  const sortByColumn = useCallback(
+    e => {
+      const column = e.target.getAttribute('data-column')
+      setTableData(sortBy(data, column))
+    },
+    [tableData]
+  )
+
+  return (
+    <div className="data-table">
```

Comment:
I wish I didn't have need to do this - I had to wrap the table to sort out some styling issues - the caption element doesn't play nice with box-shadow on a table.

Reply:
<img width="988" alt="Screenshot 2019-06-13 at 17 50 52" src="https://user-images.githubusercontent.com/48086589/59451642-d1f43780-8e03-11e9-958e-4c1ca58c34a7.png">

Reply:
@hxltrhuxze See if you can get border-radius to work on the bottom corners - I struggled with this. It seems to be an issue with `border-collapse: collapse;`.